src/lib/stitching/vortex/stitching.ts

```diff
@@ -318,17 +322,27 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
               ... on AnalyticsArtist {
                 entityId
               }
+              ... on AnalyticsViewingRoom {
+                entityId
+              }
             }
           }
         `,
         resolve: (parent, _args, context, info) => {
```

Comment:
Working on some tests for this work locally, it is proving to be very challenging 😅